app/models/fe/element.rb

```diff
@@ -119,7 +119,7 @@ def previous_element(question_sheet, page = nil)
     end
 
     def hidden_by_conditional?(answer_sheet, page)
-      prev_el = previous_element(answer_sheet.question_sheet, page)
+      prev_el = previous_element(page.question_sheet, page) if answer_sheet.question_sheets.include?(page.question_sheet)
```

Comment:
I added the guard clause.  I'm hesitant to change the method signature of previous_element (question_sheet -> array) since I don't know what all calls that method.